src/ipelib/flip/KEMPRN.f

```diff
@@ -329,7 +329,7 @@ SUBROUTINE CNOPO(J,I,JP,Z,RTS,ON,O2N,N2N,NE,P1,NOP,OPLS
 C::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
       SUBROUTINE CO2P(J,I,JP,Z,RTS,ON,O2N,N2N,NE,P1
      > ,O2P,TPROD5,OPLS,OP2D,N2PLS,NPLUS,N4S,NNO,OP2P,
-     > mp,ilp)
+     > mp,ilp,icall,its)

```

Comment:
if these new variables aren't used at all, drop them out of here and the CO2P calls in CMINOR.f